conans/client/command.py

```diff
@@ -1999,6 +2000,10 @@ def run(self, *args):
                 self._print_similar(command)
                 raise ConanException("Unknown command %s" % str(exc))
 
+            if is_config_install_scheduled(self._conan) and \
+               ((command == "config" and "install" not in args[0]) or command != "config"):
```

Comment:
what is `args[0]` in this case? check is non-intuitive, also may have false positives (for things like `uninstall`, `reinstall`)